recipes/kubernetes-client-python/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: kubernetes
```

Comment:
I actually wanted to call it kubernetes-client-python (just forgot here). This resembles the name of the official repo, and disambiguates with other packages.